package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "permitio",
-  "version": "2.1.0-rc.0",
+  "version": "2.2.0",
```

Comment:
2.1.0 instead of 2.2.0
we didn't release the non-RC 2.1.0 so it's the next logical minor version